<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactsCreateWindow</class>
 <widget class="QDialog" name="ContactsCreateWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>610</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>610</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="CreateContactBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>570</width>
     <height>440</height>
    </rect>
   </property>
   <property name="title">
    <string>Create Contact</string>
   </property>
   <widget class="QGroupBox" name="FullNameBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Full Name</string>
    </property>
    <widget class="QPlainTextEdit" name="FullNamePlainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>180</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="PhoneBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Phone</string>
    </property>
    <widget class="QPlainTextEdit" name="PhonePlainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>180</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="EmailBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Email</string>
    </property>
    <widget class="QPlainTextEdit" name="EmailPlainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>180</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="AddressBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Address</string>
    </property>
    <widget class="QPlainTextEdit" name="AddressPlainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>180</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="EventsBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>320</width>
      <height>350</height>
     </rect>
    </property>
    <property name="title">
     <string>Events</string>
    </property>
    <widget class="QTableView" name="EventsTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>300</width>
       <height>320</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>530</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="HorizontalLayout">
     <item>
      <spacer name="HorizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="CreateContactButton">
       <property name="text">
        <string>Create Contact</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
